models/automations/automation-events.mdx

```diff
@@ -104,6 +104,31 @@ For example, trigger an automation when average `loss` decreases by at least `.2
 ![Screenshot showing a run metrics change threshold automation](/images/automations/run_metrics_change_threshold_automation.png)
 </Frame>
 
+#### Run metrics z-score change
+<Note>
+Currently available only in [W&B Multi-tenant Cloud](/platform/hosting/#wb-multi-tenant-cloud).
+</Note>
+
+For **Run metrics z-score threshold met** events, the automation triggers when a metric's z-score (standard score) exceeds a specified threshold. A z-score indicates how many standard deviations a value is from the mean of all values for that metric across runs in the project.
+
+To create a z-score automation, you configure:
+1. The target z-score threshold, expressed as a positive float value (for example, 2.0).
+1. The window of logged values that determine the mean value (defaults to 30).
+1. The comparison to make:
+   - Above (trigger when performance is unusually high)
+   - Below (trigger when performance is unusually low)
+   - Both
+
+For example, trigger an automation when `accuracy` has a z-score above 2, indicating the run is performing significantly better than typical runs in the project.
```

Comment:
```suggestion
For example, trigger an automation when `accuracy` has a z-score above 2, which means the run is performing significantly better than other runs in the project.
```

Reply:
This is fine here, but the comparison is versus the mean of the window of runs in consideration, not versus all runs in the project.